test/house_test.rb

```diff
@@ -0,0 +1,64 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+  def setup
+    @room_1 = Room.new(:bedroom, 10, '13')
+    @room_2 = Room.new(:bedroom, 11, '15')
+    @room_3 = Room.new(:living_room, 25, '15')
+    @room_4 = Room.new(:basement, 30, '41')
+    @house = House.new("$400000", "123 sugar lane")
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_it_has_attributes
+    assert_equal "$400000", @house.price
+    assert_equal "123 sugar lane", @house.address
+    assert_equal [], @house.rooms
+  end
+
+  def test_it_can_add_room
+    @house.add_room(@room_1)
+    @house.add_room(@room_2)
+
+    assert_equal [@room_1, @room_2], @house.rooms
+  end
+
+  def test_if_house_can_get_rooms_from_category
+    @house.add_room(@room_1)
+    @house.add_room(@room_2)
+    @house.add_room(@room_3)
+    @house.add_room(@room_4)
+
+    assert_equal [@room_1, @room_2], @house.rooms_from_category(:bedroom)
+    assert_equal [@room_4], @house.rooms_from_category(:basement)
+    assert_equal 1900, @house.area
+  end
+
+  def test_if_can_get_house_details
+    assert_equal ({"price"=>"$400000", "address"=>"123 sugar lane"}), @house.house_details
```

Comment:
be sure to follow the interaction pattern exactly.  This method should just be called `details`, not `house_details`